tests/test-grammar-integration.cpp

```diff
@@ -12,6 +12,9 @@
 #include <cassert>
 #include <string>
 #include <vector>
+#include <json-schema-to-grammar.h>
+
+using json = nlohmann::ordered_json;
```

Comment:
This might be a fault of the way I merged the code, but we don't need these 3 lines because `json-schema-to-grammar.h` is already included, and this `using` line is already here.

Reply:
Oops sorry bad merge, fixed